view/next-project/src/components/sponsoractivities/SponsorActivitiesAddModal.tsx

```diff
@@ -108,6 +148,61 @@ export default function SponsorActivitiesAddModal(props: Props) {
           <option value={'回収済み'}>回収済み</option>
         </Select>
       </div>
+      <p className='text-black-600'>オプション</p>
+      <div className='col-span-4 w-full'>
+        <Select
+          value={data.feature}
+          onChange={(e) => {
+            if (e.target.value === 'クーポン') {
+              setFormData({ ...formData, feature: e.target.value, remark: remarkCoupon });
+            } else if (e.target.value === 'ポスター') {
+              setFormData({ ...formData, feature: e.target.value, remark: remarkElse });
+            } else {
+              setFormData({ ...formData, feature: e.target.value, remark: '' });
+            }
+          }}
+        >
+          <option value={'なし'} selected>
+            なし
+          </option>
+          <option
+            value={'ポスター'}
+            disabled={sponsorStyles[data.sponsorStyleID - 1]?.style === '企業ブース'}
+          >
+            ポスター
+          </option>
+          <option
+            value={'クーポン'}
+            disabled={sponsorStyles[data.sponsorStyleID - 1]?.style === '企業ブース'}
+          >
+            クーポン
+          </option>
+        </Select>
+      </div>
+      <p className='text-black-600'>移動距離(km)</p>
+      <div className='col-span-4 w-full'>
+        <Input
+          type='number'
+          className='w-full'
+          id={String(data.id)}
+          value={data.expense}
+          onChange={formDataHandler('expense')}
+        />
+      </div>
+      <p className='text-black-600'>交通費</p>
+      <div className='col-span-4 w-full'>
+        <p className='w-full'>{Math.round(data.expense * 11)}円</p>
+      </div>
+      <p className='text-black-600'>備考</p>
+      <div className='col-span-4 w-full'>
+        <Textarea
+          type='string'
+          className='w-full'
+          id={String(data.id)}
+          value={data.remark}
+          onChange={formDataHandler('remark')}
+        ></Textarea>
```

Comment:
実際のtextareaタグはtypeを取らないので追加しなくてok!
あと、タグに挟む子要素（children）がない場合は、こんな感じに省略できる！

```suggestion
        <Textarea
          className='w-full'
          id={String(data.id)}
          value={data.remark}
          onChange={formDataHandler('remark')}
        />
```